### PR TITLE
Get commit ID from build.sh

### PR DESCRIPTION
**Pull request by salvacorts**
Show last commit ID on perftest header

### CODE REVIEW

File: srcCpp03/perftest_publisher.cxx [hunk 1]
```diff
@@ -224,19 +224,23 @@ void perftest_cpp::PrintVersion()
...
 
         if (perftestV.revision != 0) {
-            printf(".%d", perftestV.revision);
+            fprintf(stderr, ".%d", perftestV.revision);
         }
     }
 
-    printf(" (RTI Connext DDS %d.%d.%d)\n",
+  #ifdef PERFTEST_COMMIT_ID
+    printf(" %s", PERFTEST_COMMIT_ID);
```
Comment (javmorcas) [inline srcCpp03/perftest_publisher.cxx]: I believe you missed this one.

Comment (salvacorts) [reply srcCpp03/perftest_publisher.cxx]: Done

File: build.bat [hunk 2]
```diff
@@ -363,6 +363,17 @@ if !BUILD_CPP! == 1 (
...
+	where git >nul 2>nul
+	If !ERRORLEVEL! == 0 (
+		for /f "tokens=* USEBACKQ" %%F in (`git rev-parse --short HEAD`) do (
+			set commit_id=%%F
+		)
+		
+		set "ADDITIONAL_DEFINES=!ADDITIONAL_DEFINES! PERFTEST_COMMIT_ID=\"!commit_id!\""
+	)
+
+	echo [INFO]: BYEEEEE
```
Comment (javmorcas) [inline build.bat]: LOL! Bye? I guess this is the only thing remaining, right?

**Comment by javmorcas** [comment]
No Github issue associated to this.

**Pull request merged**